core/src/Kokkos_HPX.hpp

```diff
@@ -204,6 +204,8 @@ namespace Experimental {
 class HPX {
  private:
   static bool m_hpx_initialized;
+  static bool m_was_initialized;
+  static bool m_was_finalized;
```

Comment:
Why did you elected to declare them as static here and not in the Cuda and HIP backends?